components/table/hooks/useFilter/FilterDropdown.tsx

```diff
@@ -537,15 +604,29 @@ function FilterDropdown<RecordType>(props: FilterDropdownProps<RecordType>) {
         getPopupContainer={getPopupContainer}
         placement={direction === 'rtl' ? 'bottomLeft' : 'bottomRight'}
         rootClassName={rootClassName}
+        destroyPopupOnHide
```

Comment:
The content inside `menu` is not rerendered,  so I cannot pass `visible` to FocusLock to set the `disabled` prop, as it doesn't update component, there is an only way to run the `onDeactivation` callback